composer.json

```diff
@@ -1,15 +1,42 @@
 {
-  "name": "drupal/drush-cmi-tools",
-  "type": "drupal-drush",
-  "description": "Export configuration to a directory and apply an ignore list.",
-  "keywords": [
-    "Drupal"
-  ],
-  "license": "GPL-2.0+",
-  "homepage": "https://github.com/previousnext/drush_cmi_tools",
-  "support": {
-    "issues": "https://github.com/previousnext/drush_cmi_tools/issues",
-    "source": "https://github.com/previousnext/drush_cmi_tools"
-  },
-  "require": {}
-}
\ No newline at end of file
+    "name": "drupal/drush_cmi_tools",
+    "type": "drupal-module",
+    "description": "Export configuration to a directory and apply an ignore list.",
+    "keywords": [
+        "Drupal"
+    ],
+    "license": "GPL-2.0+",
+    "homepage": "https://github.com/previousnext/drush_cmi_tools",
+    "support": {
+        "issues": "https://github.com/previousnext/drush_cmi_tools/issues",
+        "source": "https://github.com/previousnext/drush_cmi_tools"
+    },
+    "repositories": {
+        "packages.drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    },
+    "require": {
+        "drupal/core": "^8.4",
+        "drush/drush": "^9.0.0-beta6"
```

Comment:
https://github.com/previousnext/drush_cmi_tools/pull/19#issuecomment-323539649

Reply:
Currently you can't run this as standalone. Drush 9 requires this to be a module in order to use the service container.